_data/simple-icons.json

```
@@ -5591,6 +5591,11 @@
             "hex": "FF6801",
             "source": "https://www.joinhoney.com"
         },
+        {
+            "title": "Honor",
+            "hex": "000000",
+            "source": "https://www.hihonor.com/"
```

```suggestion
            "source": "https://www.hihonor.com"
```
Remove trailing slash